lib/TPP/DefaultTppPasses.cpp

```diff
@@ -138,7 +140,6 @@ struct DefaultTppPasses
             BrgemmLinalgTilingOptions{lhsTile, rhsTile}));
         pm.addNestedPass<func::FuncOp>(createLoopInvariantCodeMotionPass());
         pm.addNestedPass<func::FuncOp>(createVectorizationPass());
-        pm.addNestedPass<func::FuncOp>(createCanonicalizerPass());
```

Comment:
Any reason to remove?

Reply:
We need `brgemm tiling -> licm -> vectorization -> hoisting -> canonicalizer -> vector to fma`  order for  the `vector-to-kernel` path.

And for Kavitha's `vector-to-xsmm` path the input IR is after `vectorization` pass. So, we moved the `hoisting -> canonicalizer -> vector to fma` into VectorToKernel.cpp file.

Please note, `canonicalizer` should be after `hoisting` pass because it fuses outer tiling loops and it results in `no pattern matching` for `hoisting` pass.

Reply:
I don't like that the three passes above are not a "bundle" per se, but the idea is to remove the `linalg-to-vector` flag and always vectorize whatever `linalg-to-xsmm` didn't catch. For a later PR, of course.

Reply:
@adam-smnk Sure. Added the note.